Gemfile

```diff
@@ -3,7 +3,7 @@ gemspec
 
 gem 'rest-client', :github => 'opscode/rest-client'
 
-gem 'chef-pedant', :github => 'opscode/chef-pedant', :tag => '1.0.46'
+gem 'oc-chef-pedant', :github => 'opscode/oc-chef-pedant', :tag => '2.0.0'
```

Comment:
both `chef-pedant` and `oc-chef-pedant` were deprecated in their own repos and they now exist in the chef server repo.  Can we just use a released 2.0.0 version?